versions.properties

```diff
@@ -0,0 +1,2 @@
+gradlePlugin=3.6.2
```

Comment:
is this so that the buildsrc build.gradle can access these as well? i feel like the amount of complexity (and overhead from continually loading this file) added for that isn't worth it and we should just take the small messiness hit to define these versions in two places instead

Reply:
Yep, totally agree. The buildSrc only need to keep up with the major Android Gradle plugin version; even then unless the Tasks or Plugins are using removed APIs it will keep working, the buildSrc can drift quite a bit before it even starts to become an issue at all.